python/ray/tune/tests/test_run_experiment.py

```diff
@@ -204,7 +204,7 @@ def on_result(self, result):
                 }
             }
         })
-        self.assertTrue(
+        self.assertFalse(
```

Comment:
I thought about this and I think it makes sense to never add any logger callbacks if someone sets the `TUNE_DISABLE_AUTO_CALLBACK_LOGGERS` env variable to 1. That's a deliberate choice and we shouldn't mess with that.
Before this PR there was a reason to add this (since we distinguished between legacy `Logger` classes and callbacks), but since we're moving to all-callbacks now, there's no need to do magic here.